frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -56,14 +60,14 @@ const parseEncodedSnapshots = async (
         }
         try {
             const snapshotLine = typeof l === 'string' ? (JSON.parse(l) as EncodedRecordingSnapshot) : l
-            const snapshotData = snapshotLine['data']
+            const snapshotData = isRecordingSnapshot(snapshotLine) ? [snapshotLine] : snapshotLine['data']
 
             return snapshotData.map((d: unknown) => {
                 const snap = withMobileTransformer
                     ? postHogEEModule?.mobileReplay?.transformEventToWeb(d) || (d as eventWithTime)
                     : (d as eventWithTime)
                 return {
-                    windowId: snapshotLine['window_id'],
+                    windowId: snapshotLine['window_id'] || snapshotLine['windowId'],
```

Comment:
shouldn't be only one way since it's our protocol?

Reply:
🤔  it's this method that's normalising the windowId before export

So, the stored value has `window_id` but then this transforms it for the UI, then we export it... 🤔 

I'll do another pass and see how painful that'd be to sort out

Reply:
would be a bit of a pain to do a quick version of this...

added a comment instead

Will punt into the future

Reply:
(because I want to do the styling changes 😊)

Reply:
If there's a comment to explain why we fallback to the other option, all good.